test/con_cache_test.exs

```diff
@@ -184,6 +184,24 @@ defmodule ConCacheTest do
     end)
   end
 
+  test "get_or_store on bag" do
+    with_cache([ets_options: [:bag]],fn(cache) ->
+      assert ConCache.get_or_store(cache, :a, fn() -> 1 end) == 1
```

Comment:
Sounds good to me 👍  Once you disable `get_or_store`, we can merge this PR.

Since you opened a request for bags, I'm assuming you have some use case in mind, so perhaps that can serve as an example of how should these disabled functions behave. In any case, feel free to open issues, and we'll discuss further.

Thanks!